File: titre.php
<?php

$url_abes = "http://www.sudoc.fr/";
$rdf = ".rdf";
$rech= "dc:title";

$ppns = array('08410533X','002390248','011097809');

foreach ($ppns as $ppn) {
    $url = $url_abes . $ppn . $rdf;
    $page = file_get_contents($url);

    if ($res = preg_match("#<$rech>([^<]+)</$rech>#", $page, $matches)) {
        echo sprintf("%s\n", $matches[1]);
    }
}
 

File: titre.pl
#!/usr/bin/perl

use strict;
use warnings;
use LWP::Simple;

my $url_abes = "http://www.sudoc.fr/";
my $rdf = ".rdf";
my $rech= "dc:title";


my @ppn = ('08410533X','002390248','011097809');

foreach my $ppn (@ppn) {
 my $url = $url_abes . $ppn . $rdf;
 my $page = get($url);
   if ($page =~ /<$rech>(.+)<\/$rech>/) {
     print "$1\n";
     }
   }
 

